system-parachains/asset-hubs/asset-hub-polkadot/src/xcm_config.rs

```diff
@@ -67,6 +70,11 @@ parameter_types! {
 	pub const GovernanceLocation: MultiLocation = MultiLocation::parent();
 	pub RelayTreasuryLocation: MultiLocation = (Parent, PalletInstance(polkadot_runtime_constants::TREASURY_PALLET_ID)).into();
 	pub TreasuryAccount: AccountId = TREASURY_PALLET_ID.into_account_truncating();
+	// Test [`crate::tests::treasury_pallet_account_not_none`] ensures that the result of location
+	// conversion is not `None`.
+	pub RelayTreasuryPalletAccount: AccountId =
+		LocationToAccountId::convert_location(&RelayTreasuryLocation::get())
+			.unwrap_or(TreasuryAccount::get());
```

Comment:
We still have the "old" `TreasuryAccount`. What's the difference between the two? Shouldn't we remove the old one and make sure everything uses the correct account?

Reply:
I didn't see before that it uses `TreasuryAccount` as a fallback 🤦 . IMO that is reasonable, it could always be rescued by Root if something went there.